src/Hyperwallet/Hyperwallet.php

```diff
@@ -417,6 +419,27 @@ public function createTransfer(Transfer $transfer) {
         return new Transfer($body);
     }
 
+    /**
+     * Create a transfer refund
+     *
+     * @param Transfer $transfer The transfer data
+     * @return Transfer
+     *
+     * @throws HyperwalletArgumentException
+     * @throws HyperwalletApiException
+     */
+    public function createTransferRefund(TransferRefund $transferRefund) {
+        if (empty($transferRefund->getToken())) {
+            throw new HyperwalletArgumentException('transferToken is required!');
+        }
+        if (empty($transferRefund->getClientRefundId())) {
+            throw new HyperwalletArgumentException('clientRefundId is required!');
+        }
+
+        $body = $this->client->doPost('/rest/v3/transfers/{transfer-token}/refunds', array(), $transferRefund, array());
```

Comment:
Please write an Integration test and verify it. urlParams should be passed correctly.

Reply:
Done. Removed the non-functional method. This was implemented in the branch HW-58098-PHP-SDK-Transfer-refunds